pom.xml

```diff
@@ -264,6 +265,12 @@
       <version>${awaitility.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>${assertj-core.version}</version>
```

Comment:
this is not related to the pr but, this library is amazing. fluent + easy assertions.

Reply:
I think if we go with this library then we should move all assertions into using that (into another PR). That could be fun.

Reply:
it's a small change. i'll do it immediately after this one.